### PR TITLE
Added solution for 26th August 

### PR DESCRIPTION
**Pull request by AnjaliPatle**

### CODE REVIEW

File: August_26_fizz-buzz.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Solution {
+public
```
Comment (priyanshisharma) [inline August_26_fizz-buzz.cpp:2]:
```suggestion
public:
```

File: August_26_fizz-buzz.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Solution {
+public
+    vectorstring fizzBuzz(int n) {
```
Comment (priyanshisharma) [inline August_26_fizz-buzz.cpp:3]:
```suggestion
    vector<string> fizzBuzz(int n) {

```

File: August_26_fizz-buzz.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class Solution {
+public
+    vectorstring fizzBuzz(int n) {
+        vectorstring s;
+        for(int i=1; in+1; i++){
+            if(i%3==0 && i%5==0){
+                s.push_back(FizzBuzz);
```
Comment (priyanshisharma) [inline August_26_fizz-buzz.cpp:7]:
```suggestion
                s.push_back("FizzBuzz");

```

File: August_26_fizz-buzz.cpp [hunk 4]
```diff
@@ -0,0 +1,20 @@
+class Solution {
+public
+    vectorstring fizzBuzz(int n) {
+        vectorstring s;
+        for(int i=1; in+1; i++){
+            if(i%3==0 && i%5==0){
+                s.push_back(FizzBuzz);
+            }else if(i%3==0){
+                s.push_back(Fizz);
```
Comment (priyanshisharma) [inline August_26_fizz-buzz.cpp:9]:
```suggestion
                s.push_back("Fizz");

```

File: August_26_fizz-buzz.cpp [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+public
+    vectorstring fizzBuzz(int n) {
+        vectorstring s;
+        for(int i=1; in+1; i++){
+            if(i%3==0 && i%5==0){
+                s.push_back(FizzBuzz);
+            }else if(i%3==0){
+                s.push_back(Fizz);
+            }else if(i%5==0){
+                s.push_back(Buzz);
```
Comment (priyanshisharma) [inline August_26_fizz-buzz.cpp:11]:
```suggestion
                s.push_back("Buzz");

```

**Review by priyanshisharma — CHANGES_REQUESTED** [review]
Hey @AnjaliPatle I'm really glad to see your PR, but I've already assigned this to @sai-pothuri, please refer #1 I will definitely merge your PR if he doesn't come up with a solution in some time ✨ 

Untill then, there are some small changes required in your code to make it work, please check them out 😄

**Comment by AnjaliPatle** [comment]
Hey @priyanshisharma no problem at all! I did not see the assignment before contributing 😅
Also, thanks for pointing out the errors, must have happend while changing file formats. I've committed the changes you suggested.

**Review by priyanshisharma — APPROVED** [review]
Thanks @AnjaliPatle , I see the other PR is made in Python and hence merging this as well 🙌

**Pull request merged**